### ISSUE TITLE
Explore adding functionality to create more lines for specific rows in the form

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/simple-annoton-editor#85]
From: https://github.com/geneontology/noctua/issues/583

From @suzialeksander 
I need to add two "has_input"s to an annotation. Is there a way to tweak the Form? I managed to do this through the Graph Editor but it wasn't intuitive (two different titles for what, to the user, are the same thing: "sub-nodes"/"edge & class expression"

From @vanaukenk 
Perhaps we could add a feature to the form that would allow curators to add another row of contextual information, although we will want to be selective about which rows could be duplicated.

**Comment by vanaukenk** [comment]
Note that multiple 'has input' relations and values are allowed in the current spec:

https://github.com/geneontology/GO_Shapes/blob/master/shapes/go-cam-shapes.shex

**Comment by tmushayahama** [comment]
@suzialeksander @vanaukenk yes, this feature will be implemented on Form 2.0 . I am still playing around with how to best expand the form without overwhelming it on its default ( first impressions :) ) @thomaspd had a good suggestion of adding the option on the MF row like below

![image](https://user-images.githubusercontent.com/2273929/62185231-7976ec00-b316-11e9-9deb-6fc7408de000.png)

 With proper labeling of course. However hiding it like this, means noone will know it's there unless they dig in. Any thoughts?

**Comment by vanaukenk** [comment]
Thanks @tmushayahama 

I agree that we need to make this functionality more visible.  

How about something simple like a '+' sign next to the rows that can have more than one entry (according to the shex specifications, of course)?

Clicking on the '+' sign would generate an additional row.

**Comment by pgaudet** [comment]
Out of date.
